src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountContent.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff.content.countycourtjudgment;
+
+public class AmountContent {
+
+    private String totalAmount;
+    private String interestAmount;
+    private String interestDailyAmount;
+    private String feeAmount;
+    private String paidAmount;
+    private String remainingAmount;
+
+    public AmountContent(
+        String totalAmount,
```

Comment:
If we say that content is having responsibility in preparing data for rendering then it should take correct type, BigDecimal in this case` and do formatting to pound-strings inside.

Reply:
It's a bit inconsistent, I've had the content provider format and I think that is ok